src/main/java/org/opensearch/agent/tools/SearchIndexTool.java

```diff
@@ -48,7 +48,7 @@ public class SearchIndexTool implements Tool {
 
     public static final String TYPE = "SearchIndexTool";
     private static final String DEFAULT_DESCRIPTION =
-        "Use this tool to search an index by providing two parameters: 'index' for the index name, and 'query' for the OpenSearch DSL formatted query.";
+        "Use this tool to search an index by providing two parameters: 'index' for the index name, and 'query' for the OpenSearch DSL formatted query. Only use this tool when a DSL query is available.";
```

Comment:
Have you tested with this description, will LLM still generates DSL automatically and then invoke this tool? Or this tool will only be invoked when user DSL is provided?

Reply:
I have tested 8 user cases from tool selection. None of the case will trigger search index tool.